examples/osgi-test-example-bnd/itest/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+-runrequires.junit: \
+	bnd.identity;id='junit-platform-launcher',\
+	bnd.identity;id='junit-jupiter-engine'
+
+# Build dependencies 
+-buildpath: \
+	${junit},\
+	${mockito},\
+	${osgitest},\
+	osgi.annotation;version='7.0',\
+	osgi.core;version='6.0',\
+	osgi.cmpn;version='7.0',\
+	junit-jupiter-api;version='5.7.0'
+	
+javac.source: 11
+javac.target: 11
+
+# We need JUnit and Mockito to resolve the test cases at runtime. 
+# Other runtime dependencies should be added as necessary
+-runbundles: \
+	org.apache.felix.configadmin;version='[1.9.16,1.9.17)',\
+	junit-jupiter-api;version='[5.7.0,5.7.1)',\
+	junit-platform-commons;version='[1.7.0,1.7.1)',\
+	org.opentest4j;version='[1.2.0,1.2.1)',\
+	org.osgi.test.common;version='[0.10.0,0.10.1)',\
+	org.osgi.test.junit5;version='[0.10.0,0.10.1)',\
+	junit-platform-engine;version='[1.7.0,1.7.1)',\
+	junit-platform-launcher;version='[1.7.0,1.7.1)',\
+	org.osgi.test.junit5.cm;version='[0.10.0,0.10.1)',\
+	itest;version=snapshot,\
+	junit-jupiter-engine;version='[5.7.0,5.7.1)'
+
+
+Bundle-Version: 0.0.0.${tstamp}
+Private-Package: itest
+
+# Needed for Mockito's mocking to work
+-runsystempackages.objenesis: sun.misc,sun.reflect
+
+# Use Felix by default
+-runfw: org.apache.felix.framework;version='[6.0.1,6.0.1]'
```

Comment:
Why hard code version when only one version exists in the repo?

Reply:
removed version